miniapp/USERGUIDE.md

```diff
@@ -116,8 +116,11 @@ CoroutineScope(Dispatchers.IO).launch {
 
 The `MiniAppMessageBridge` is used for passing messages between the Mini App (JavaScript) and the Host App (your native Android App) and vice versa. Your App must provide the implementation for these functions and pass this implementation to the `MiniApp#create` function.
 
+There are some methods have default implementation but host app can override them to fully control.
```

Comment:
I think this is okay for now, but in the future we will need to reformat this section about `MiniAppMessageBridge`. I think it will get kind of long and confusing as we add more and more features 😅

Reply:
Updated.